examples/17-shared-model/src/Shared/Model.elm

```diff
@@ -0,0 +1,20 @@
+module Shared.Model exposing (Model, WindowSize)
+
+{-| -}
+
+
+{-| Normally, this value would live in "Shared.elm"
+but that would lead to a circular dependency import cycle.
+
+For that reason, both `Shared.Model` and `Shared.Msg` are in their
+own file, so they can be imported by `Effect.elm`
+
+-}
+type alias Model =
+    { windowSize : WindowSize }
+
+
+type alias WindowSize =
+    { width : Int
+    , height : Int
+    }
```

Comment:
I think it would be nice to keep things simple and directly add `windowWidth` and `windowHeight` to Model. Developer may choose how to encapsulate.

Reply:
> I think it would be nice to keep things simple and directly add `windowWidth` and `windowHeight` to Model. Developer may choose how to encapsulate.

Point taken. Thanks.